src/cgame/cg_main.cpp

```diff
@@ -1198,7 +1198,7 @@ static void SetMapInfoFromArenaInfo( arenaInfo_t arenaInfo )
 
 static void SetMapInfo()
 {
-	std::string mapName = Cvar::GetValue( "mapname" );
```

Comment:
There's `cgs.mapname` you can use

Reply:
The map loading screen is painted before `cgs.mapname` is set and we better want to have the name displayed as soon as possible to not see the map load screen for some time without text, then see the text appearing.

Reply:
For reference the code to display the levelshot also uses `Info_ValueForKey( CG_ConfigString( CS_SERVERINFO ), "mapname" )`, I assume it's also to display it as soon as possible:

https://github.com/Unvanquished/Unvanquished/blob/756925e9ec1a4d31e680511423f2ba3609057ba0/src/cgame/cg_rocket_draw.cpp#L1797